core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java

```diff
@@ -198,26 +199,35 @@ public void onEvent(InetUtils.IPChangeEvent event) {
      * @return update is success
      */
     public boolean update(Member newMember) {
+        return update(newMember, new ObjectEqual<Member>() {
+            @Override
+            public boolean equals(Member obj1, Member obj2) {
+                return MemberUtils.fullEquals(obj1, obj2);
+            }
+        });
+    }
+    
+    public boolean update(Member newMember, ObjectEqual<Member> objEqual) {
         Loggers.CLUSTER.debug("member information update : {}", newMember);
-        
+    
         String address = newMember.getAddress();
         if (!serverList.containsKey(address)) {
             return false;
         }
-        
+    
         serverList.computeIfPresent(address, (s, member) -> {
             if (NodeState.DOWN.equals(newMember.getState())) {
                 memberAddressInfos.remove(newMember.getAddress());
             }
-            if (!MemberUtils.fullEquals(newMember, member)) {
+            if (!objEqual.equals(newMember, member)) {
                 newMember.setExtendVal(MemberMetaDataConstants.LAST_REFRESH_TIME, System.currentTimeMillis());
                 MemberUtils.copy(newMember, member);
                 // member data changes and all listeners need to be notified
                 NotifyCenter.publishEvent(MembersChangeEvent.builder().members(allMembers()).build());
             }
             return member;
         });
-        
+    
```

Comment:
Please do not change the indent.

Please use nacos codestyle to reformat your change.

detail see source code `style/nacosCodeStyle.md.`

Reply:
OK